pytest_mh/utils/services.py

```diff
@@ -46,7 +46,8 @@ def async_start(self, service: str) -> SSHProcess:
         self.__set_initial_state(service)
         self.logger.info(f'systemd: starting "{service}" asynchronously')
         return self.host.ssh.async_run(
-            f'systemctl start "{service}" || systemctl status "{service}"', log_level=SSHLog.Error
+            f'systemctl reset-failed "{service}" && systemctl start "{service}" || systemctl status "{service}"',
```

Comment:
Use `;` instead of `&&`. `reset-failed` may fail in the unit is stopped and not enabled by default (this is triggered by autofs tests in sssd). Using `;` is fine, if the service fails to start and `reset-failed` failed, we will see it in the logs. And if by any chance `reset-failed` failed but start succeeded, we don't care.

Same for restart.

Reply:
Agreed.